### DISCUSSION TITLE
How can I use AlphaTexImporter ?

### DISCUSSION THREAD

**Discussion by claroja** [CoderLine/alphaTab#695] [Q&A] [answered]
Thanks for your great job! Maybe it's the best project for guitar players.
I have read the article https://www.alphatab.net/docs/guides/lowlevel-apis , but I want to use AlphaTexImporter to get same result. The problem is I can't import AlphaTexImporter in my code. How can I fix it? Thank you.

**Comment by Danielku15** [comment, marked as answer]
The individual importers are not exposed but you can simply use the ScoreLoader. It also will try loading the data using the AlphaTexImporter.  https://www.alphatab.net/docs/guides/lowlevel-apis#loading-files-via-scoreloader

Just use TextEncoder to convert your string to an Uint8Array.

It might eat a bit CPU to try all Importers but it should he insignificant. Maybe I can expose some better way to give hints to the ScoreLoader to avoid crawling them all and also add some AlphaTex loader to the ScoreLoader. 

Until then you can try the workaround above.

**Reply by claroja** [reply]
Thanks very much! It works.
